src/pages/developing/angular-tutorial/step-5.mdx

```diff
@@ -152,7 +152,7 @@ first.
 npm run build
 ```
 
-Looking at `package.json`, you'll find `yarn build` to run `ng build`. This
+Looking at `package.json`, you'll find `ng build`. This
```

Comment:
Just curious why the switch away from `yarn build`? `npm run build` also seems a better choice

Reply:
Sure, in the tutorial, this is the only mention to yarn.